config/app_config.go

```diff
@@ -0,0 +1,104 @@
+	TimeoutToken                    string
+	TimeoutDefault                  string
+	EnableRequestLog                bool
+	EnablePrintRequest              bool
+	ElasticURL                      string
+	TokenSafeDurationInMinutes      int
+	RetryNumberGetBoleto            int
+	InfluxDBHost                    string
+	InfluxDBPort                    string
+	RecoveryRobotExecutionEnabled   string
+	RecoveryRobotExecutionInMinutes string
+	EnableMetrics                   bool
+	EnableFileServerCertificate     bool
+	WaitSecondsRetentationLog       string
+	ForceTLS                        bool
+}
+
+type QueueConfig struct {
+	ConnQueue                        string
+	OriginExchange                   string
+	OriginQueue                      string
+	OriginRoutingKey                 string
+	TimeToRecoveryWithQueueInSeconds string
+	Heartbeat                        string
+	QueueMaxTLS                      string
+	QueueMinTLS                      string
+	QueueByPassCertificate           bool
+}
+
+func getServerConfig() ServerConfig {
+	return ServerConfig{
+		APIPort:                         ":" + os.Getenv("API_PORT"),
+		PdfAPIURL:                       os.Getenv("PDF_API"),
+		Version:                         os.Getenv("API_VERSION"),
+		MachineName:                     getHostName(),
+		ApplicationName:                 "BoletoOnline",
+		BuildVersion:                    os.Getenv("BUILD_VERSION"),
+		Environment:                     os.Getenv("ENVIRONMENT"),
+		AppURL:                          os.Getenv("APP_URL"),
+		MaskLogs:                        os.Getenv("MASK_LOGS") == "true",
+		TimeoutRegister:                 os.Getenv("TIMEOUT_REGISTER"),
+		TimeoutToken:                    os.Getenv("TIMEOUT_TOKEN"),
+		TimeoutDefault:                  os.Getenv("TIMEOUT_DEFAULT"),
+		EnableRequestLog:                os.Getenv("ENABLE_REQUEST_LOG") == "true",   // Log a cada request no SEQ
+		EnablePrintRequest:              os.Getenv("ENABLE_PRINT_REQUEST") == "true", // Imprime algumas informacoes da request no console
+		ElasticURL:                      os.Getenv("ELASTIC_URL"),
+		TokenSafeDurationInMinutes:      getValueInt(os.Getenv("TOKEN_SAFE_DURATION_IN_MINUTES")),
+		RetryNumberGetBoleto:            getValueInt(os.Getenv("RETRY_NUMBER_GET_BOLETO")),
+		InfluxDBHost:                    os.Getenv("INFLUXDB_HOST"),
+		InfluxDBPort:                    os.Getenv("INFLUXDB_PORT"),
+		RecoveryRobotExecutionEnabled:   os.Getenv("RECOVERYROBOT_EXECUTION_ENABLED"),
+		RecoveryRobotExecutionInMinutes: os.Getenv("RECOVERYROBOT_EXECUTION_IN_MINUTES"),
+		EnableMetrics:                   os.Getenv("ENABLE_METRICS") == "true",
+		EnableFileServerCertificate:     os.Getenv("ENABLE_FILESERVER_CERTIFICATE") == "true",
+		WaitSecondsRetentationLog:       os.Getenv("WAIT_SECONDS_RETENTATION_LOG"),
+		ForceTLS:                        strings.ToLower(os.Getenv("FORCE_TLS")) == "true",
+	}
+}
+
+func getQueueConfig() QueueConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀 
```suggestion
func getQueueConfig() *QueueConfig {
```